code/game/machinery/cloning.dm

```diff
@@ -545,11 +547,17 @@ GLOBAL_LIST_INIT(cloner_biomass_items, list(\
 
 /obj/machinery/clonepod/update_icon()
 	..()
-	icon_state = "pod_0"
+	cut_overlays()
+
+	if(panel_open)
+		add_overlay(list("panel_open"))
```

Comment:
Can be passed as a non list. The code in add_overlay will do the creating of the list for you.

Reply:
Hm so it can.. will change.